### PR TITLE
Add an example to deploy Redis cluster on GKE.

### PR DESCRIPTION
**Pull request by shunjikawabata**

### CODE REVIEW

File: examples/redis-cluster-gke/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## How to use
+
+1. Provision a GKE cluster.
+
+```
+$ gcloud container clusters create redis-cluster --num-nodes 6 --machine-type n1-standard-8 --image-type COS --disk-type pd-standard --disk-size 100 --enable-ip-alias --create-subnetwork name=redis-subnet
+```
+
+Please specify your own project ID or prefered zone if you asked.
```
Comment (sabhyankar) [inline examples/redis-cluster-gke/README.md]: Perhaps you can reword that to:
If prompted, please specify your preferred project id and zone.

File: examples/redis-cluster-gke/configmaps/redis-conf.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```
Comment (sabhyankar) [inline examples/redis-cluster-gke/configmaps/redis-conf.yaml]: Add Apache 2.0 license header

File: examples/redis-cluster-gke/redis-cache.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+apiVersion: policy/v1beta1
```
Comment (sabhyankar) [inline examples/redis-cluster-gke/redis-cache.yaml]: Add Apache 2.0 license header

File: examples/redis-cluster-gke/configmaps/redis-expect.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```
Comment (sabhyankar) [inline examples/redis-cluster-gke/configmaps/redis-expect.yaml]: Add Apache 2.0 header.

File: examples/redis-cluster-gke/redis-create-cluster.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+apiVersion: batch/v1
```
Comment (sabhyankar) [inline examples/redis-cluster-gke/redis-create-cluster.yaml]: Add Apache 2.0 header.

File: examples/redis-cluster-gke/README.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
+# Deploying Redis Cluster on GKE
+
```
Comment (sabhyankar) [inline examples/redis-cluster-gke/README.md]: Add a prerequisites section:
1. Gcloud SDK, kubectl, git etc
2. Access to a GCP project and using an id with appropriate privileges.

File: examples/redis-cluster-gke/README.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+$ kubectl get po
+NAME                           READY   STATUS      RESTARTS   AGE
+redis-cache-55dfdcb84c-4tzh4   1/1     Running     0          5m
+redis-cache-55dfdcb84c-79kkn   1/1     Running     0          5m
+redis-cache-55dfdcb84c-98gn9   1/1     Running     0          5m
+redis-cache-55dfdcb84c-t4gc6   1/1     Running     0          5m
+redis-cache-55dfdcb84c-ttg85   1/1     Running     0          5m
+redis-cache-55dfdcb84c-xdbz8   1/1     Running     0          5m
+redis-create-cluster-cpjsp     0/1     Completed   0          3m
+```
```
Comment (sabhyankar) [inline examples/redis-cluster-gke/README.md]: Can we add a section for validation? Perhaps connect via redis-cli or link to instructions that demonstrate how to connect via redis-cli?

File: examples/redis-cluster-gke/README.md (hunk 6 above)
Comment (shunjikawabata) [reply examples/redis-cluster-gke/README.md]: Added pre-requisites.

File: examples/redis-cluster-gke/README.md (hunk 7 above)
Comment (shunjikawabata) [reply examples/redis-cluster-gke/README.md]: Added validation steps.

File: examples/redis-cluster-gke/configmaps/redis-conf.yaml (hunk 2 above)
Comment (shunjikawabata) [reply examples/redis-cluster-gke/configmaps/redis-conf.yaml]: Added Apache license header.

File: examples/redis-cluster-gke/configmaps/redis-expect.yaml (hunk 4 above)
Comment (shunjikawabata) [reply examples/redis-cluster-gke/configmaps/redis-expect.yaml]: Added Apache license header.

File: examples/redis-cluster-gke/redis-cache.yaml (hunk 3 above)
Comment (shunjikawabata) [reply examples/redis-cluster-gke/redis-cache.yaml]: Added Apache license header.

File: examples/redis-cluster-gke/redis-create-cluster.yaml (hunk 5 above)
Comment (shunjikawabata) [reply examples/redis-cluster-gke/redis-create-cluster.yaml]: Added Apache license header.

File: examples/redis-cluster-gke/README.md (hunk 1 above)
Comment (shunjikawabata) [reply examples/redis-cluster-gke/README.md]: Modified.

**Pull request merged**